src/configuration.ts

```diff
@@ -46,14 +46,27 @@ export const getUseDebugLogs = (): boolean | undefined => {
     return useDebugLogs;
 };
 
-export const getCheckovVersion = (): string => {
+export const getCheckovVersion = async (logger: Logger): Promise<string> => {
 
     const configuration: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration('checkov');
     const checkovVersion = configuration.get<string>('checkovVersion', 'latest').trim().toLowerCase();
 
     if (checkovVersion === '' || checkovVersion === 'latest') {
         return 'latest';
+    } else if (checkovVersion === 'platform') {
+        logger.debug('Checkov version is "platform" - getting version from platform API');
+
+        const token = configuration.get<string>('token');
+        const prismaApiUrl = configuration.get<string>('prismaURL');
+        if (!token) {
```

Comment:
The checkovVersion endpoint does not require authentication. I think this step can be skipped